### PR TITLE
CAKE-5234 Affiliate link disclaimer

### PR DESCRIPTION
**Pull request by mklucsarits**
## Links
* http://wikia-inc.atlassian.net/browse/CAKE-5234

## Description
Add a disclaimer on the article page and search results page for affiliate units appearing on those pages.

## Reviewers
@Wikia/cake

### CODE REVIEW

File: public/locales/en/main.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "affiliate-unit": {
-    "big-unit-heading": "Suggested For You"
+    "big-unit-heading": "Suggested For You",
+    "disclaimer": "Disclosure: Some of the links above are affiliate links meaning, at no additional cost to you, Fandom will earn a commission if you click through and make a purchase."
```
Comment (cschucode) [inline public/locales/en/main.json]: Noticed that the copy in the ticket didn't have a period. Was that a typo in the ticket?

> Disclosure: Some of the links above are affiliate links meaning, at no additional cost to you, Fandom will earn a commission if you click through and make a purchase

**Comment by cschucode** [comment]
Padding seems to be off on Search Results Page

Screen Shot 2019-11-04 at 10 46 23 AM

**Comment by JoshuaRogan** [comment]
@cschucode beat me to it!

image

File: app/styles/component/_affiliate-unit.scss [hunk 2]
```diff
@@ -66,3 +66,13 @@
     margin: 0 .75rem;
   }
 }
+
+.affiliate-unit-disclaimer {
+  font-size: $wds-font-size-xs;
+  line-height: 1.25;
```
Comment (vforge) [inline app/styles/component/_affiliate-unit.scss]: There's variable for that

File: app/templates/components/wikia-ui-components/wikia-card-crosswiki-list.hbs [hunk 3]
```diff
@@ -11,7 +11,11 @@
     </li>
     {{#if includePostsForQuery}}
       {{#if (equal key 1)}}
-        {{post-search-interrupt query=includePostsForQuery isCrossWiki=true}}
+        {{post-search-interrupt
+          query=includePostsForQuery
```
Comment (JoshuaRogan) [inline app/templates/components/wikia-ui-components/wikia-card-crosswiki-list.hbs:15]: Out of curiosity is there any linting rule or standard when to do this?

Comment (mklucsarits) [reply app/templates/components/wikia-ui-components/wikia-card-crosswiki-list.hbs:15]: I don't think there's a linting rule, it just felt like the right thing to do. The line seemed too long when I added the additional attribute.

File: app/styles/component/_affiliate-unit.scss (hunk 2 above)
Comment (mklucsarits) [reply app/styles/component/_affiliate-unit.scss]: Fixed

**Comment by mklucsarits** [comment]
@cschucode and @JoshuaRogan Thanks, I've fixed the spacing on the search page.

**Review by JoshuaRogan — APPROVED** [review]
LGTM

File: public/locales/en/main.json (hunk 1 above)
Comment (mklucsarits) [reply public/locales/en/main.json]: I added the period, but I'll remove it just in case.

**Comment by mklucsarits** [comment]
I've tested everything together after merging in the dev branch -- new targeting, the query parameter and the disclaimer -- and it all looks good.

**Pull request merged**